### DISCUSSION TITLE
SQL changes

### DISCUSSION THREAD

**Discussion by JosephSuvak** [JosephSuvak/QR-Ticket-Generator#9] [General]
For discussing the Database structure

**Comment by JohnBanas** [comment]
If you pull from the current develop branch `acd55df` you can:

1.  seed the database and log in as a user 
2. create a new user 
3. log out your current session 
4. access the concert information 
5. access the user's tickets.

**Comment by JosephSuvak** [comment]
Please note that these credentials are not permanent.
Heroku rotates credentials periodically and updates applications where this database is attached.

**Host**
ec2-3-233-7-12.compute-1.amazonaws.com

**Database**
dbllhfqf0omnce

**User**
qawbctgejktgok

**Port**
5432

**Password**
8c1e9334a67bc5a324aa7be2899c1b4e8bccd656e11bc5841ce1d0bdfd2b414f

**URI**
postgres://qawbctgejktgok:8c1e9334a67bc5a324aa7be2899c1b4e8bccd656e11bc5841ce1d0bdfd2b414f@ec2-3-233-7-12.compute-1.amazonaws.com:5432/dbllhfqf0omnce

**Heroku CLI**
heroku pg:psql postgresql-perpendicular-04617 --app qrater-app

**Comment by JosephSuvak** [comment]
To request access to this, contact me to be added to the Heroku development environment.
